app/records-interfaces/group-records-interface.coffee

```diff
@@ -1,5 +1,3 @@
-null
```

Comment:
this might all break once we minify for production, but whatevs we can leave that till then.